### PR TITLE
Add jq.yml to check if a json is sorted

### PR DESCRIPTION
**Pull request by Freed-Wu**
See <https://github.com/Freed-Wu/nerdfont.vim/actions/runs/3308994392/jobs/5461765978>

If a json is unsorted, test will fail.

Screenshot from 2022-10-24 07-34-06

### CODE REVIEW

File: .github/workflows/jq.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  pull_request:
+    paths:
+      - '**.json'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker://nanozoo/jq:1.6--0bafc15
```
Comment (lambdalisue) [inline .github/workflows/jq.yml]: You don't need this. jq is already installed in GitHub hosted runner.

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: .github/workflows/jq.yml (hunk 1 above)
Comment (Freed-Wu) [reply .github/workflows/jq.yml]: I have tested you are right, I'll fix it. I am curious where you know this? Does github have any manual about it?

Comment (lambdalisue) [reply .github/workflows/jq.yml]: Yes. It's listed on GitHub's official documentation for GitHub hosted runner. I lost the URL thought you can find it from google.

Comment (Freed-Wu) [reply .github/workflows/jq.yml]: OK I found it <https://docs.github.com/cn/actions/using-github-hosted-runners/about-github-hosted-runners#preinstalled-software>

**Pull request closed without merging**